src/System.ComponentModel.Annotations/src/Configurations.props

```diff
@@ -1,13 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project DefaultTargets="Build">
   <PropertyGroup>
    	<PackageConfigurations>
       netstandard;
-      net461;
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations)
-      netfx;
```

Comment:
Do we want to stop building the sources agaisnt netfx? That means that the tests now will run against the netstandard implementation, rather than testing the typeforwards and netfx inbox implementation

Reply:
Maybe add a placeholder configuration here? `_netfx`?